src/index.html

```diff
@@ -1,12 +1,455 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#logo">
+        <img
+          src="/images/logo.png"
+          alt="logo.png">
+      </a>
+
+      <nav class="nav">
+        <div class="nav__list">
+          <a class="nav__link is-active" href="#aplle">apple
+          </a>
+
+          <a class="nav__link"
+            href="#samsung">samsung
+          </a>
+
+          <a class="nav__link"
+            href="#smartphones">smartphones
+          </a>
+
+          <a
+            class="nav__link"
+            href="#laptops & computers"
+            data-qa="nav-hover"
+          >laptops & computers
+          </a>
+
+          <a class="nav__link"
+            href="#gadgets">gadgets
+          </a>
+
+          <a class="nav__link"
+            href="#tablets">tablets
+          </a>
+
+          <a class="nav__link"
+            href="#photo">photo
+          </a>
+
+          <a class="nav__link"
+            href="#video">video
+          </a>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac.jpeg"
+            class="card__image"
+          >
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+
+          </div>
+
+          <div class="card__price-container">
+            <p class="card__price-label">
+              Price:
+            </p>
+
+            <p class="card__price">
+              $2,199
+            </p>
+          </div>
```

Comment:
It's not two paragraph, it's only one. (Приклад: "ціна 2000 грн", це не два різні параграфа, це по суті одне речення, не можна написати: "Ціна.  2000 грн.", бо це одне речення, отак і з параграфами)
```suggestion
          <p class="card__price-container">
            <span class="card__price-label">
              Price:
            </span>

            <span class="card__price">
              $2,199
            </span>
          </p>
```

Reply:
again semantic mistakes... fixed and updated!